src/components/m-button/index.js

```diff
@@ -6,8 +8,23 @@ import Button from './js/button';
 
 class AXAButton extends BaseComponentGlobal {
   static tagName = 'axa-button'
+  static propTypes = {
+    arrow: PropTypes.bool,
+    classes: PropTypes.string,
+    color: PropTypes.oneOf(['red', 'white']),
+    ghost: PropTypes.bool,
+    motion: PropTypes.bool,
+    size: PropTypes.oneOf(['sm', 'lg']),
+    tag: PropTypes.oneOf(['a', 'button']),
+    href: PropTypes.string,
```

Comment:
All URL-like props like `href`, `url`, `src`, `uri`, etc should have a distinct check for URLs

Reply:
agree

Reply:
unfortunately I haven't found a good URL prop type checker yet.
We could use `URL` API because it throws for wrong URLs, but it's polyfill is rather large `1.8kB` gziped 😲